libraries/AP_HAL_Linux/RCOutput_Bebop.cpp

```diff
@@ -283,6 +283,28 @@ void RCOutput_Bebop::_play_sound(uint8_t sound)
     _dev->get_semaphore()->give();
 }
 
+void RCOutput_Bebop::play_note(uint8_t pwm, uint16_t period, uint16_t duration)
```

Comment:
Ok, I'll move RCOutput_Bebop::play_note to ToneAlarm.

About pwm:
For i2c message :
* if pwm is < 3, then pwm refers to a predefined song.
* if pwm is > 3, then pwm is really a pwm and the following arguments (period and duration are read)